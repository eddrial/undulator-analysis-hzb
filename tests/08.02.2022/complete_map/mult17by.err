 error report for the data set mult17by.dvm        
--------------------------------------------
      
         127   data points thrown away to get a
         monotonically increasing data set
 the truncated data set has only   1591 data points 
        out of totally   2558 data points
 the rms value of the residuals is  0.37791E-02
 the maximum residual is  0.13864E-01
 the rms value of the lambda/2-filtered data is  0.35324E-01
 the maximum value of the lambda/2-filtered data is  0.28321E+00
