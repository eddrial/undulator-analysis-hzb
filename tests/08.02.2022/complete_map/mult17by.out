mult17by.dvm
            

 names of parameters as follows : 
b(1) == amplitude, 1st harm.
b(2) == amplitude, 3rd harm.
b(3) == amplitude, 5rd harm.
....                        
b(n) == amplitude, n^th harm.
b(n+1) == lambda, mm.
b(n+2) == x0, 1st harm, mm.
b(n+3) == x0, 3rd harm, mm.
b(n+4) == x0, 5th harm, mm.
....
b(nparm) == x0, n^th harm, mm.

 parameters:
   1  9.9275935E-01
   2 -9.5425034E-03
   3 -8.7507414E-03
   4 -1.2537603E-04
   5 -2.1494791E-04
   6  5.5980759E+01
   7  1.0438895E+03
   8  1.0436643E+03
   9  1.0439583E+03
  10  4.8822340E-01
  11 -4.1539742E+01
 number of iterations:   16
     sumsq:  2.27221E-02
       rms:  3.77911E-03  dimension of magnetic field
       rms:  3.80665E-01  percent of b_eff
   rms_ext:  4.03699E-01  percent of b_eff
 B_eff =   9.9277E-01
 B_max =   9.7484E-01
 K_eff =    5.193
 lamda_1 (nm) =     36.78673

        eigenvectors in parameter space

 error                         direction
     1.36,  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  1.00  0.00
     0.62,  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  1.00
     0.04,  0.00  0.00  0.00  0.00  0.00  0.00  0.00  1.00  0.00  0.00  0.00
     0.03,  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  1.00  0.00  0.00
     0.00,  0.00  0.00  0.00  0.00  0.00 -0.05  1.00  0.00  0.00  0.00  0.00
     0.00,  0.65 -0.45 -0.42 -0.30 -0.31 -0.11 -0.01  0.00  0.00  0.00  0.00
     0.00,  0.50  0.52  0.57 -0.37 -0.12  0.00  0.00  0.00  0.00  0.00  0.00
     0.00,  0.49  0.40 -0.31  0.68  0.19  0.01  0.00  0.00  0.00  0.00  0.00
     0.00,  0.23 -0.30  0.14 -0.17  0.90 -0.03  0.00  0.00  0.00  0.00  0.00
     0.00,  0.17 -0.52  0.62  0.52 -0.21 -0.06  0.00  0.00  0.00  0.00  0.00
     0.00,  0.09 -0.09  0.00 -0.01 -0.02  0.99  0.05  0.00  0.00  0.00  0.00
   
  calculated kicks for trajectory compensation: 
 kick_1:  -0.27943 Tmm
 kick_2:   0.27943 Tmm
      
 the rms value of the second derivative is  0.00000E+00
 the maximum value of the second derivative is  0.00000E+00
      
 the maximum transverse movement of the electron in mu_meter
                  (peak to peak) is  0.31435E+02
      
 the truncated data set has only   1591 data points 
      
 the rms value of the residuals is  0.37791E-02
 the maximum residual is  0.13864E-01
      
 the rms value of the lambda/2-filtered data is  0.35324E-01
 the maximum value of the lambda/2-filtered data is  0.28321E+00
      
 the rms phase error (all data) is    2.714 deg.
 the maximum phase error (all data) is    7.288 deg.
      
 the rms phase error (extrema) is    3.084 deg.
 the maximum phase error (extrema) is    8.245 deg.
      
 the Walker-rms phase error (all data) is   19.122 deg.
 the Walker-maximum phase error (all data) is   33.128 deg.
      
 the Walker-rms phase error (extrema) is    3.312 deg.
 the Walker-maximum phase error (extrema) is    8.069 deg.
  
  phase: straight line subtracted
 -----------------------------------------
 total phase error             2.713524    
 local phase error             2.056204    
 slowly varying phase error    1.582630    
 oscillatory phase error      0.7940537    
 -----------------------------------------
